config/tutorials/gh-archive/kafka-source.yaml

```diff
@@ -1,6 +1,8 @@
 version: 0.6
 source_id: kafka-source
 source_type: kafka
+max_num_pipelines_per_indexer: 2
```

Comment:
Can you set it to 1 as generally it's better to start with this default value?

Reply:
Updated